spec/spec_helper.rb

```diff
@@ -1,100 +1,27 @@
-# This file was generated by the `rspec --init` command. Conventionally, all
-# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
-# The generated `.rspec` file contains `--require spec_helper` which will cause
-# this file to always be loaded, without a need to explicitly require it in any
-# files.
-#
-# Given that it is always loaded, you are encouraged to keep this file as
-# light-weight as possible. Requiring heavyweight dependencies from this file
-# will add to the boot time of your test suite on EVERY test run, even for an
-# individual file that may not need all of that loaded. Instead, consider making
-# a separate helper file that requires the additional dependencies and performs
-# the additional setup, and require it from the spec files that actually need
-# it.
-#
+# frozen_string_literal: true
+
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Why is this necessary? 
It can happen when the file is loaded and the `RAILS_ENV` is not 'test'?

Reply:
I think you can run easily with other environment.

I was thinking about adding some kind of db truncate (I am not sure yet how we should build the whole testing env) and it could be some safety stuff with `abort 'Running in prod env' if Rails.env.production?` to prevent production db truncation.